tests/__tests__/console/ConsoleTable_test.re

```diff
@@ -0,0 +1,428 @@
+    );
+    expect.string(
+      Table.createElement(
+        ~columns=[{width: 7}, {width: 4}],
+        ~children=[
+          Row.createElement(~children=["testing", "1234"], ()),
+          Row.createElement(~children=["a"], ()),
+        ],
+        (),
+      ),
+    ).
+      toEqual(
+      ""
+      ++ "------------------\n"
+      ++ "| testing | 1234 |\n"
+      ++ "|---------+------|\n"
+      ++ "| a       |      |\n"
+      ++ "------------------",
+    );
+  });
+
+  test("Text wrapping", ({expect}) => {
+    let actual =
+      Table.createElement(
+        ~columns=[{width: 7}, {width: 4}],
+        ~children=[
+          Row.createElement(~children=["test a long column", "this"], ()),
+          Row.createElement(~children=["a", "b"], ()),
+        ],
+        (),
+      );
+    let expected =
+      ""
+      ++ "------------------\n"
+      ++ "| test a  | this |\n"
+      ++ "| long    |      |\n"
+      ++ "| column  |      |\n"
+      ++ "|---------+------|\n"
+      ++ "| a       | b    |\n"
+      ++ "------------------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Basic table JSX", ({expect}) => {
+    let actual =
+      <Table columns=[<ColumnConfig width=7 />, <ColumnConfig width=4 />]>
+        <Row> "testing" "1234" </Row>
+        <Row> "a" "b" </Row>
+      </Table>;
+    let expected =
+      ""
+      ++ "------------------\n"
+      ++ "| testing | 1234 |\n"
+      ++ "|---------+------|\n"
+      ++ "| a       | b    |\n"
+      ++ "------------------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Large table", ({expect}) => {
+    let actual =
+      <Table
+        columns=[
+          <ColumnConfig width=8 />,
+          <ColumnConfig width=16 />,
+          <ColumnConfig width=8 />,
+        ]>
+        <Row> "a1" "b1" "c1" </Row>
+        <Row> "a2" "b2 is a cell that needs to wrap" "c2" </Row>
+        <Row> "a3" "b3" "c3" "d3" </Row>
+        <Row> "a4" "b4" </Row>
+        <Row> "a5" "b5" "c5" </Row>
+      </Table>;
+    let expected =
+      ""
+      ++ "------------------------------------------\n"
+      ++ "| a1       | b1               | c1       |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a2       | b2 is a cell     | c2       |\n"
+      ++ "|          | that needs to    |          |\n"
+      ++ "|          | wrap             |          |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a3       | b3               | c3       |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a4       | b4               |          |\n"
+      ++ "|----------+------------------+----------|\n"
+      ++ "| a5       | b5               | c5       |\n"
+      ++ "------------------------------------------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Formatted Table", ({expect}) => {
+    let bold_cell = Pastel.(<Pastel bold=true> "b1" </Pastel>);
```

Comment:
This is going to fail in CI, you either want to wrap this entire body in Pastel.useMode(Terminal) or make a Pastel copy and set it to terminal 

(e.g.)
```reason
module TerminalPastel = Pastel.Make({});
TerminalPastel.setMode(Terminal);
```

and then use `<TerminalPastel>` everywhere

Reply:
I think it's worth having some e2e Terminal tests, but I think explicitly using HumanReadable mode for some tests would be useful as well. This can be done the same way as the other example by replacing Terminal with HumanReadable